source/MaterialXCore/Document.cpp

```diff
@@ -1061,6 +1061,45 @@ void Document::upgradeVersion(bool applyFutureUpdates)
                     }
                 }
             }
+
+            // Make it so that interface names and nodes in a nodegraph are not duplicates
+            // If they are, rename the nodes.
+            for (NodeGraphPtr nodegraph : getNodeGraphs())
+            {
+                std::set<string> interfaceNames;
+                for (ElementPtr child : nodegraph->getChildren())
+                {
+                    NodePtr srcNode = child->asA<Node>();
```

Comment:
Super nitpick, feel free to ignore: Maybe rename `srcNode` to just `node` for consistency, and since it has nothing to do with being a "source" here.